src/diffusers/models/embeddings.py

```diff
@@ -391,15 +391,16 @@ def get_3d_rotary_pos_embed(
         The size of the temporal dimension.
     theta (`float`):
         Scaling factor for frequency computation.
-    use_real (`bool`):
-        If True, return real part and imaginary part separately. Otherwise, return complex numbers.
 
     Returns:
         `torch.Tensor`: positional embedding with shape `(temporal_size * grid_size[0] * grid_size[1], embed_dim/2)`.
     """
+    if use_real is not True:
```

Comment:
I removed the support for use_real for now since it is not used in cogvideox (and the logic is somewhat different from what I would have expected based on our 2d rope implementation - let's add that support in the future when it is needed)